joblib/hashing.py

```diff
@@ -93,39 +88,7 @@ def memoize(self, obj):
             return
         Pickler.memoize(self, obj)
 
-    # The dispatch table of the pickler is not accessible in Python
-    # 3, as these lines are only bugware for IPython, we skip them.
-    def save_global(self, obj, name=None, pack=struct.pack):
```

Comment:
Not sure what this IPython-specific code is for. Is that causing a problem if you keep it like this?